### PR TITLE
feat(common): Interface

### PR DESCRIPTION
**Pull request by benak66**
Create ui-server interface types

### CODE REVIEW

**Comment by Sciator** [comment]
Solves #8

File: common/interface/ad.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import { TCommonData, TCommonDataNew } from "./common";
+
+export enum TAdCategory {
```
Comment (Sciator) [inline common/interface/ad.ts:3]: TAdCategory is enum so it should be named EAdCategory

File: common/interface/common.ts [hunk 2]
```diff
@@ -12,3 +19,151 @@ export type TErrorResponse = {
...
+  name: string,
+  description: string,
+  place?: TPlace,
+  photo?: TPhoto,
+  tags?: TTag[],
+};
+
+// Common Actions
+export type TGenericResponse = {
+  wasError: boolean,
```
Comment (Sciator) [inline common/interface/common.ts:69]: redundant. remove this. We know, that response resulted in error because error isn't undefined

**Comment by kejmUP** [comment]
Closing, replaced with Kotlin data classes and typescript-generator plugin.

**Pull request closed without merging**